src/shapes/text.class.js

```diff
@@ -212,6 +212,14 @@
      */
     startOffset:               0,
 
+    /**
+     * Which side of the path the text should be drawn on.
+     * Only used when text has a path
+     * @type {String} 'left|right'
+     * @default
+     */
+    side:               'left',
```

Comment:
for the previous discussion, should we call this `pathSide`?

Reply:
sounds good. I think let's change `startOffset` to `pathStartOffset` as well for the same reason

Reply:
yes i agree is fresh enough that no one noticed it yet.